source/extensions/common/wasm/wasm.cc

```diff
@@ -175,8 +175,11 @@ Word resolve_dns(void* raw_context, Word dns_address_ptr, Word dns_address_size,
     root_context->onResolveDns(token, status, std::move(response));
   };
   if (!context->wasm()->dnsResolver()) {
+    envoy::config::core::v3::DnsResolutionConfig dns_resolution_config;
+    envoy::config::core::v3::TypedExtensionConfig dns_resolver_config;
+
     context->wasm()->dnsResolver() = context->wasm()->dispatcher().createDnsResolver(
-        {}, envoy::config::core::v3::DnsResolverOptions());
+        dns_resolution_config, dns_resolver_config);
```

Comment:
In the new implementation an array of resolvers will be created (whereas in the previous one an empty array is being sent). Is this change intentional?

Reply:
ACK